apps/frontend/src/libs/hooks/use-app-form/use-app-form.hook.ts

```diff
@@ -24,6 +25,7 @@ type ReturnValue<T extends FieldValues = FieldValues> = {
 	errors: FieldErrors<T>;
 	handleSubmit: UseFormHandleSubmit<T>;
 	setError: UseFormSetError<T>;
+	watch: UseFormWatch<T>;
```

Comment:
we can also add .refine method on a zod validation schema to check if passwords are the same

Reply:
Initially, I did this way, but under my previous pr, I got message to handle it while submitting, not in validation schema